riscv/triggers.cc

```diff
@@ -22,7 +22,6 @@ reg_t mcontrol_t::tdata1_read(const processor_t * const proc) const noexcept {
   v = set_field(v, MCONTROL_CHAIN, chain_bit);
   v = set_field(v, MCONTROL_MATCH, match);
   v = set_field(v, MCONTROL_M, m);
-  v = set_field(v, MCONTROL_H, h);
```

Comment:
Shouldn't the `MCONTROL_H` define be removed too?

Reply:
In encoding.h? encoding.h is out of date wrt the debug spec in numerous ways, and I wasn't planning on fixing that.

I'm planning to move this code to use debug_defines.h, which is auto-generated from the same source that creates the debug spec, so is always correct.